pennylane/qchem/structure.py

```diff
@@ -191,7 +193,7 @@ def active_space(electrons, orbitals, mult=1, active_electrons=None, active_orbi
     return core, active
 
 
-def excitations(electrons, orbitals, delta_sz=0):
+def excitations(electrons, orbitals, delta_sz=0, fermionic=False):
```

Comment:
Add information about `fermionic` keyword in the documentation.

Reply:
Yes. The `Return` part needs to be updated as well since it can also return a FermiWord now.

Reply:
It's updated in the last commit.